Applications/DataExplorer/NetCdfDialog/NetCdfConfigureDialog.cpp

```diff
@@ -158,17 +138,16 @@ int NetCdfConfigureDialog::setVariableSelect()
 {
     int max_dim = 0;
     int max_dim_idx = 0;
-    auto const n_vars = _currentFile->num_vars();
-    for (int i = 0; i < n_vars; i++)
+    auto const& names =_currentFile.getVars();
+    for (auto it = names.cbegin(); it != names.cend(); ++it)
```

Comment:
Could be
```suggestion
    for (auto [xxx, yyy] : names)
```
and later just use `xxx` and `yyy` instead of `it->first/second`. Choose proper names, of course.

Reply:
That syntax doesn't work for me for some reason.

FWIW, "names" is a multimap.